ntoskrnl/mm/ARM3/miarm.h

```diff
@@ -1002,7 +1002,6 @@ MI_WRITE_INVALID_PTE(IN PMMPTE PointerPte,
 {
     /* Write the invalid PTE */
     ASSERT(InvalidPte.u.Hard.Valid == 0);
-    ASSERT(InvalidPte.u.Long != 0);
```

Comment:
@tkreuzer : So, is it expected that this field can sometimes be != 0, but also sometimes be == 0 ?

Reply:
@HBelusca, If you look at the following part of the regression test, The would trigger this ASSERT unnecessaryily.
```
    /* PoolTag = 0 */
    Mapping = MmAllocateMappingAddress(1, 0);
    ok(Mapping == NULL, "MmAllocateMappingAddress failed\n");
    if (Mapping != NULL)
    {
        MmFreeMappingAddress(Mapping, 0);
    }
    ```

Reply:
I added it to catch accidental zeroing, when it wasn't intended. But in this case 0 is a legitimate value. And I don't think this ASSERT is really that helpful.